domain/markets/models/src/main/kotlin/com/tangem/domain/markets/TokenMarketParam.kt

```diff
@@ -1,35 +1,32 @@
-package com.tangem.features.markets.details
+package com.tangem.domain.markets
 
 import com.tangem.domain.core.serialization.SerializedBigDecimal
-import com.tangem.domain.markets.TokenMarket
 import kotlinx.serialization.Serializable
 
 @Serializable
-data class TokenMarketSerializable(
+data class TokenMarketParam(
```

Comment:
Вроде везде используем Params.
nit:
```suggestion
data class TokenMarketParams(
```